### PR TITLE
Reupload failed reports to scout

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR does two things:
1. reduces the number of analyses to upload delivery report for
1. sets the analysis.delivery_reported_at date AFTER the report uploaded to scout

Contact: @emiliaol 

How to test reduction in number of analyses to upload report for:
- [x] install master on stage on hasta 
- [x] login to clinical-db
- [x] run `dbcopyprodtostage.sh cg` on clinical-db
- [x] run `cg upload delivery-reports` and note the number of reports processed 
- [x] install this branch on stage on hasta
- [x] run `dbcopyprodtostage.sh cg` on clinical-db
- [x] run `cg upload delivery-reports` and note the number of reports processed

Expected outcome:
- [x] The number of reports being processed in the second run was lower.

How to test that the analysis does not get report date when uploaded to scout unsuccessfull:
- [x] install on hasta stage
- [x] run `dbcopyprodtostage.sh cg` on clinical-db
- [x] run `dbcopyprodtostage.sh housekeeper` on clinical-db
- [x] find a analysis that has uploaded_at and delivered_at and delivery_reported_at dates and exists in scout
- [x] remove the  delivery_reported_at date
- [x] remove delivery-report from housekeeper
- [x] remove case from scout
- [x] run `cg upload delivery-report CASE_ID`

Expected outcome:
- [x] The report should not get a delivery_reported_at date

How to test that the analysis does not get report date when uploaded to scout unsuccessfull:
- [x] install on hasta stage
- [x] run `dbcopyprodtostage.sh cg` on clinical-db
- [x] run `dbcopyprodtostage.sh housekeeper` on clinical-db
- [x] find a analysis that has uploaded_at and delivered_at and delivery_reported_at dates and exists in scout
- [x] remove the  delivery_reported_at date
- [x] run `cg upload delivery-report CASE_ID`

Expected outcome:
- [x] The report should get a delivery_reported_at date

Reviewers:
- [x] code review by KB
- [x] Tested by @patrikgrenfeldt 
- [x] Deploy OK:d by 

- [x] This is a minor **version bump** because it expands the interface in a backwards compatible manner
- [ ] Deployed

### CODE REVIEW

File: cg/cli/upload.py [hunk 1]
```diff
@@ -153,12 +153,11 @@ def delivery_report(context, family_id, print_console):
 
         if added_file:
             click.echo(click.style('uploaded to housekeeper', fg='green'))
-            context.invoke(delivery_report_to_scout, case_id=family_id)
-            click.echo(click.style('uploaded to scout', fg='green'))
-            _update_delivery_report_date(status_api, family_id)
         else:
             click.echo(click.style('already uploaded to housekeeper, skipping'))
 
+        context.invoke(delivery_report_to_scout, family_id=family_id)
```
Comment (ingkebil) [inline cg/cli/upload.py]: Oh man! Nice catch :)

**Comment by ingkebil** [comment]
Code reviewed and looking good! :iphone:

**Comment by patrikgrenfeldt** [comment]
image

The report should not get a delivery_reported_at date:
image

The report should get a delivery_reported_at date:
image

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**